grpc/streambuffer/streambuffer.go

```diff
@@ -126,7 +139,7 @@ func (s *Stream) Run() (err error) {
 			if s.recvFunc != nil {
 				r = s.recvFunc()
 			} else {
-				r = new(empty.Empty)
+				r = new(empty.Empty) // Generic proto message if not interested in received values
```

Comment:
If `recvFunc` is not set, the value is completely ignored, so I don't think that is relevant